phac-nml/irida-next
# Refactor distance of time

## What does this PR do and why?
This PR is refactoring the use of the rails function ```distance_of_time_in_words``` into a helper function ```time_ago``` to include an html title attribute of the date the action was performed (created/updated date, etc.), so that the actual date is available for the user upon hovering. The ```time_ago``` helper uses an existing ```viral tooltip``` component and can now be called in the views pages for when ```distance_of_time_in_words``` would be used so that all views will contain this date on hover functionality.

## Screenshots or screen recordings
![image](https://github.com/phac-nml/irida-next/assets/82407232/fb5c6c4b-07dd-47d1-a393-2648f422b1cd)


## How to set up and validate locally
1. Check the following pages and hover over the time ago element to check if the tooltip contains the expected date. Create the associated element if needed (group, project, members, etc.).
- Projects and Groups: Dashboard, Group Links, Members
- Sample Attachments

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
